### PR TITLE
fix: 修复小尺寸的input文字没有居中的问题

### PR DESCRIPTION
**Pull request by lmsccc**
[中文模板 / Chinese Template](https://github.com/DouyinFE/semi-design/blob/main/.github/PULL_REQUEST_TEMPLATE.zh-CN.md)

- [ ] I have read and followed [Pull Request Guidelines](https://github.com/DouyinFE/semi-design/blob/main/CONTRIBUTING-en-US.md#pull-request-guidelines) of the contributing guide.

### What kind of change does this PR introduce? (check at least one)

 - [x] Bugfix
 - [ ] Feature
 - [ ] Code style update
 - [ ] Refactor
 - [ ] Test Case
 - [ ] TypeScript definition update
 - [ ] Document improve
 - [ ] CI/CD improve
 - [ ] Branch sync
 - [ ] Other, please describe:

### PR description

- 修复input组件在小尺寸状态下，文字没有很居中的问题，略微向下偏了一点点，改动涉及到input基础样式，看到大部分的衍生样式都把display设置成inline-flex，所以是否可以wrapper默认的display也设置成inline-flex？这样可以解决input没有在父容器居中给的问题

### Changelog
🇨🇳 Chinese
- Fix: 修复input在small size 状态下，文字略微向下偏移的问题

---

🇺🇸 English
- Fix: Fixed the problem that the text was slightly offset downwards when the input was in small size

### Checklist
- [x] Test or no need
- [x] Document or no need
- [x] Changelog or no need

### Other
- [ ] Skip Changelog

### CODE REVIEW

File: packages/semi-foundation/input/input.scss [hunk 1]
```diff
@@ -32,7 +32,7 @@ $module: #{$prefix}-input;
 
 
 .#{$module}-wrapper {
-    display: inline-block;
+    display: inline-flex;
```
Comment (YyumeiZhang) [inline packages/semi-foundation/input/input.scss:35]: 看了下，比起修改 display，看起来修改 font-size 更合理？ default 模式是 14px，反而 small 和 large 的 size 是 16px

Comment (lmsccc) [reply packages/semi-foundation/input/input.scss:35]: 嗯嗯，确实修改字号影响可能影响会更小，但如果把字号加大了，看起来就不那么"small“了，而且也会破坏已有的表现

Comment (lmsccc) [reply packages/semi-foundation/input/input.scss:35]: 不对，small的字号应该是14呀？large是16，default是14

Comment (YyumeiZhang) [reply packages/semi-foundation/input/input.scss:35]: image

Comment (lmsccc) [reply packages/semi-foundation/input/input.scss:35]: 确实，那我修改下

**Review by YyumeiZhang — APPROVED** [review]

**Pull request merged**